pkg/helpers/helpers.go

```diff
@@ -24,6 +25,11 @@ func GetIngestrDestinationType(asset *pipeline.Asset) (pipeline.AssetType, error
 }
 
 func PrefixGenerator() string {
+	// Always return same when testing
+	if flag.Lookup("test.v") != nil {
+		return "abcefghi"
+	}
```

Comment:
why not create another function and use that for the tests instead?

Reply:
This was quicker and required less changes in the code. I thought it was smart